xikolo-ios/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -31,6 +33,15 @@ class CourseListViewController : AbstractCourseListViewController {
         updateView()
     }
 
+    override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
+        switch traitCollection.horizontalSizeClass {
+        case UIUserInterfaceSizeClass.Compact, UIUserInterfaceSizeClass.Unspecified:
```

Comment:
Is `UIUserInterfaceSizeClass` required here, i.e. is `case .Compact, .Unspecified:` not enough?